.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @redhat-appstudio/release-team will be requested for
+# review when someone opens a pull request.
+*       @redhat-appstudio/release-team
```

Comment:
Is it with or without `@`?

Reply:
```
# Teams can be specified as code owners as well. Teams should
# be identified in the format @org/team-name. Teams must have
# explicit write access to the repository.
```

Reply:
I was just commenting because what Scott said in the channel. That the `@` should not be there. I guess it only applies to the UI and not to the config file.

Reply:
Yep. The `@` sign is also used when mentioning the whole team in a comment. Only the Reviewers and Assignees don't use it.